doc/manual.rst

```diff
@@ -6333,6 +6333,24 @@ It's also possible to use ``from module import nil`` if one wants to import
 the module but wants to enforce fully qualified access to every symbol
 in ``module``.
 
+By using the experimental feature ``typeImports`` it's possible to bring
+all the symbols directly related to a type. Any callable that has the imported
+type as one of its arguments or return type gets automatically included with the
+import.
+
+.. code-block:: nim
+    :test: "nim c $1"
+
+  {.experimental: "typeImports".}
+
+  from colors import Color
+
+  # to string and proc imported
+  let pink = rgb(255, 192, 203)
```

Comment:
I think this example shows that it's not enough the type is the return type of ``rgb``.

Reply:
sorry, I don't understand, could you rephrase?

Maybe it should demonstrate what's going on more explicitly `let pink: Color = rgb(255, 192, 203)`?

Reply:
I mean that ``rgb`` should not be imported implicitly as the connection to ``Color`` is invisible.